<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ControlWindow</class>
 <widget class="QMainWindow" name="ControlWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>990</width>
    <height>616</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BotFather</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../res/gui.qrc">
    <normaloff>:/gui/black_on_trans.png</normaloff>:/gui/black_on_trans.png</iconset>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AllowTabbedDocks|QMainWindow::AnimatedDocks</set>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTextEdit" name="log_text">
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="placeholderText">
       <string>Nothing here so far...</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="save_button">
        <property name="text">
         <string>Save</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="clear_button">
        <property name="text">
         <string>Clear</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>990</width>
     <height>23</height>
    </rect>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::PreventContextMenu</enum>
   </property>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>He&amp;lp</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionAboutQt"/>
   </widget>
   <widget class="QMenu" name="menuAccount">
    <property name="title">
     <string>Acco&amp;unt</string>
    </property>
    <addaction name="actionPremiumPlans"/>
    <addaction name="actionLogout"/>
   </widget>
   <addaction name="menuHelp"/>
   <addaction name="menuAccount"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::PreventContextMenu</enum>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <property name="floatable">
    <bool>true</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionStart"/>
   <addaction name="actionStop"/>
   <addaction name="actionKill"/>
   <addaction name="separator"/>
   <addaction name="actionScripts"/>
   <addaction name="actionSettings"/>
   <addaction name="separator"/>
   <addaction name="actionBrowser"/>
   <addaction name="actionAndroid"/>
  </widget>
  <action name="actionStart">
   <property name="icon">
    <iconset resource="../res/gui.qrc">
     <normaloff>:/gui/icons8-play-64.png</normaloff>:/gui/icons8-play-64.png</iconset>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
   <property name="toolTip">
    <string>Start the bot</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../res/gui.qrc">
     <normaloff>:/gui/icons8-shutdown-64.png</normaloff>:/gui/icons8-shutdown-64.png</iconset>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
   <property name="toolTip">
    <string>Stop the bot</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="icon">
    <iconset resource="../res/gui.qrc">
     <normaloff>:/gui/icons8-settings-filled-64.png</normaloff>:/gui/icons8-settings-filled-64.png</iconset>
   </property>
   <property name="text">
    <string>Settings</string>
   </property>
   <property name="toolTip">
    <string>Open the settings dialog</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>&amp;About Botfather</string>
   </property>
  </action>
  <action name="actionScripts">
   <property name="icon">
    <iconset resource="../res/gui.qrc">
     <normaloff>:/gui/icons8-page-64.png</normaloff>:/gui/icons8-page-64.png</iconset>
   </property>
   <property name="text">
    <string>Scripts</string>
   </property>
  </action>
  <action name="actionKill">
   <property name="icon">
    <iconset resource="../res/gui.qrc">
     <normaloff>:/gui/icons8-cancel-64.png</normaloff>:/gui/icons8-cancel-64.png</iconset>
   </property>
   <property name="text">
    <string>Kill</string>
   </property>
   <property name="toolTip">
    <string>Kill the script</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionAboutQt">
   <property name="text">
    <string>About &amp;Qt</string>
   </property>
  </action>
  <action name="actionBrowser">
   <property name="icon">
    <iconset resource="../res/gui.qrc">
     <normaloff>:/gui/icons8-web-design-64.png</normaloff>:/gui/icons8-web-design-64.png</iconset>
   </property>
   <property name="text">
    <string>Browser</string>
   </property>
   <property name="toolTip">
    <string>Opens botfathers integrated browser</string>
   </property>
  </action>
  <action name="actionAndroid">
   <property name="icon">
    <iconset resource="../res/gui.qrc">
     <normaloff>:/gui/icons8-android-64.png</normaloff>:/gui/icons8-android-64.png</iconset>
   </property>
   <property name="text">
    <string>Android</string>
   </property>
   <property name="toolTip">
    <string>Open Android Info</string>
   </property>
  </action>
  <action name="actionLogout">
   <property name="text">
    <string>&amp;Logout</string>
   </property>
  </action>
  <action name="actionPremiumPlans">
   <property name="text">
    <string>Premium Plans</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../res/gui.qrc"/>
 </resources>
 <connections/>
</ui>
